### DISCUSSION TITLE
Choosing the Windows Manager in the login screen

### DISCUSSION THREAD

**Discussion by Conkernel** [neutrinolabs/xrdp#2966] [General]
Hi,

I'd like to know if there's any chance for me to configure my xrdp files in order to be able to select/choose what windows manager I wan to use in the xrdp login screen, or maybe another place, but while connecting to the xrdp host. I know we can configure some parameters in the **sesman.ini** file and create another entry... but I don't know how to tell xrdp which **startwm.sh** xrdp should use depending on the entry you choosed in the login menu.

Any help would be ver appreciated. Thanks in advance!!

**Comment by matt335672** [comment]
@Conkerel - at present there's no way to do this I'm afraid, although it has been requested. See #2530.
